src/duke/Storage.java

```diff
@@ -0,0 +1,131 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Storage {
+    String filepath;
+    File file;
+
+    Storage(String path) {
+        this.filepath = path;
+        this.file = new File(path);
+    }
+
+    /**
+     * Creates a new file if it does not exist.
+     * @return boolean
```

Comment:
What does the return value mean for create()? May specify it? Same for some other JavaDoc on methods.